src/centos/manifest.json

```diff
@@ -18,6 +18,23 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "architecture": "arm64",
+              "dockerfile": "src/centos/7/arm64v8",
+              "os": "linux",
+              "osVersion": "centos7",
+              "tags": {
+                "centos-7-arm64v8-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {},
+                "centos-7-arm64v8": {
+                  "isLocal": true
+                }
```

Comment:
Not needed since there is no dependent Dockerfile that needs to reference this tag.